lib/cryptoexchange/exchanges/gopax/market.rb

```diff
@@ -0,0 +1,8 @@
+module Cryptoexchange::Exchanges
+  module Gopax
+    class Market
+      NAME = 'gopax'
+      API_URL = 'https://api.gopax.co.kr/'
```

Comment:
maybe keep slash out to `API_URL` like `https://api.gopax.co.kr` and add to the one need this like
```
#{Cryptoexchange::Exchanges::Gopax::Market::API_URL}/trading-pairs/#{market_pair.base}-#{market_pair.target}/ticker
```